frontend/src/metabase/dashboard/dashboard.js

```diff
@@ -1126,6 +1131,19 @@ const dashcards = handleActions(
         [id]: { ...state[id], ...attributes, isDirty: true },
       }),
     },
+    [SET_MULTIPLE_DASHCARD_ATTRIBUTES]: {
+      next: (state, { payload: dashcards }) => {
+        const nextState = { ...state };
+        dashcards.forEach(({ id, attributes }) => {
+          nextState[id] = {
+            ...state[id],
+            ...attributes,
+            isDirty: true,
+          };
+        });
+        return nextState;
```

Comment:
Also nit-pick: could this be a reduce?

Reply:
`reduce` is usually used to _reduce_ list items into a single value. E.g. for getting a sum of all numbers in a list. Here we need to go over every _object_ value and assign them `attributes` and `isDirty`. It's closer to a `map` but applied to objects. There might be an `underscore` function for that probably

Reply:
Makes sense. I misread what you were iterating in the `.forEach`, thought it was `nextState`.